README.md

```diff
@@ -121,6 +128,13 @@ Supported domains: BLAS
     - Intel(R) Processor Graphics GEN9
     - NVIDIA(R) TITAN RTX(TM) (Not tested with other NVIDIA GPU families and products.)
 
+#### Windows*
+- CPU
+    - Intel Atom(R) Processors
+    - Intel(R) Core(TM) Processor Family
+    - Intel(R) Xeon(R) Processor Family
+- Accelerators
+    - Intel(R) Processor Graphics Gen9
```

Comment:
Does it make sense to join Linux and Windows Hardware since there is not so much difference between them? Just mark NVIDIA is supported on Linux only

Reply:
Please, check updated version